skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -874,5 +874,45 @@ def test_from_dict_to_dict_roundtrip(self):
         self.assertIs(d1[42], d2[42])
 
 
+class TestIterPositions(unittest.TestCase):
+    def setUp(self):
+        self.msa_empty = TabularMSA(())
+        self.msa = TabularMSA((DNA('ACGT'), DNA('TGCA')))
+        self.expected = [[Sequence('A'), Sequence('T')],
```

Comment:
This isn't exactly what we would expect.

`iter_positions` should return `[Sequence('AT'), Sequence('CG'), Sequence('GC'), Sequence('TA')]`

I was working on a method on `Sequence` which would concatenate different sequence objects together with their associated metadata, but I've been sidetracked by Sphinx documentation.